### PR TITLE
solution

### PR DESCRIPTION
**Pull request by yelnikov-andrii**
- [DEMO LINK](https://yelnikov-andrii.github.io/layout_catalog_hovers/)

### CODE REVIEW

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -14,9 +14,65 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    &:hover {
+      animation-duration: 300ms;
```
Comment (AnastasiyaSopotnytska) [inline src/styles/_catalog.scss:19]: Move this to element. 
Animation will not be smooth when mouse leave element.

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -14,9 +14,57 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```
Comment (Anna-Tkachenko) [inline src/styles/_catalog.scss:17]: You should specify properties that can be changeable

Comment (Anna-Tkachenko) [reply src/styles/_catalog.scss:17]: Don't use 'all' as transition-property

File: src/styles/_catalog.scss [hunk 3]
```diff
@@ -14,9 +14,57 @@
...
+    transition-property: all;
+    transform: scale(1);
+
+    &:hover {
+      color: #34568b;
```
Comment (Anna-Tkachenko) [inline src/styles/_catalog.scss:22]: Create variables for colors

File: src/styles/_catalog.scss [hunk 4]
```diff
@@ -14,9 +14,57 @@
...
+      transition-duration: 300ms;
```
Comment (Anna-Tkachenko) [inline src/styles/_catalog.scss:23]: It's redundant here

File: src/styles/_catalog.scss [hunk 5]
```diff
@@ -14,9 +14,57 @@
...
   }
 
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
+
+  .nav__item {
+    position: relative;
+  
+    &:hover {
```
Comment (Anna-Tkachenko) [inline src/styles/_catalog.scss:36]: Styles for nav should be in nav.scss file

File: src/styles/_header.scss [hunk 6]
```diff
@@ -37,5 +37,37 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
```
Comment (Anna-Tkachenko) [inline src/styles/_header.scss:40]: You should specify properties that can be changeable

File: src/styles/_header.scss [hunk 7]
```diff
@@ -37,5 +37,37 @@
...
+        bottom: 0;
+        left: 0;
+        height: 4px;
+        width: 100%;
+        background: #00acdc;
+        border-radius: 8px;
+        transition-duration: 300ms;
+        animation-name: move;
+        animation-duration: 300ms;
+      }
```
Comment (Anna-Tkachenko) [inline src/styles/_header.scss:58]: it's redundant here, you should only change text color for nav__link, check the task

File: src/styles/_header.scss [hunk 8]
```diff
@@ -37,5 +37,37 @@
...
+
+@keyframes move {
+
+  from {
+    transform: scaleX(0);
+  }
+
+  to {
+    transform: scaleX(1);
   }
```
Comment (Anna-Tkachenko) [inline src/styles/_header.scss:72]: It's redundant, use only correct transition for your card and change transform property on hover state

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

**Review by denys-cheporniuk — CHANGES_REQUESTED** [review]
Screenshot 2022-04-11 at 18 50 45
fix all comments from Anna

**Review by denys-cheporniuk — APPROVED** [review]

**Pull request closed without merging**